### PR TITLE
Final review #1

### PR DESCRIPTION
**Pull request by Dineshg49**

### CODE REVIEW

File: experiment/theory.md [hunk 1]
```diff
@@ -1 +1,5 @@
-### Link your theory in here
\ No newline at end of file
+- In a impact test a specially prepared notched specimen is fractured by a single blow from a heavy hammer and energy required being a measure of resistance to impact. Impact load is produced by a swinging of an impact weight W (hammer) from a height h.
```
Comment (raj-vlabs) [inline experiment/theory.md]: Why should these points be in a list?
Also check for grammatical mistakes here.

**Review by raj-vlabs — CHANGES_REQUESTED** [review]

File: experiment/simulation/js/main.js [hunk 2]
```diff
@@ -1 +1,247 @@
...
+
+
+    function restart() {
+        window.clearTimeout(tmHandle);
+        setAll();
+        play();
+    }
+
+    function play() {
+        if (flag1 === 0) {
```
Comment (raj-vlabs) [inline experiment/simulation/js/main.js]: What do flag1 and flag2 represent? Is it possible to give them meaningful names?

**Review by raj-vlabs — CHANGES_REQUESTED** [review]

File: experiment/simulation/js/main.js (hunk 2 above)
Comment (Dineshg49) [reply experiment/simulation/js/main.js]: they mark the end of some part of the simulation.

File: experiment/pretest.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                "d": "To obtain absolute values"
+            },
+            "explanations": {
+                "b": "The principal or main reason that the impact test is conducted upon a material is to ascertain if the material has undergone the required accurate heat treatment. The experiment gives no absolute value."
+            },
+            "correctAnswer": "b",
+            "difficulty": "intermediate"
+        },
+        {
+            "question": "In an Izod test after the pendulum is released from a certain height, it hits the material at the __________ point of its path.",
```
Comment (raj-vlabs) [inline experiment/pretest.json]: This question is the same as the last question in posttest. Surprisingly both have different answers.
Is this approved by the SME?

File: experiment/simulation/js/main.js (hunk 2 above)
Comment (raj-vlabs) [reply experiment/simulation/js/main.js]: Yes so I asked if they can be given more meaningful names which capture the fact that they mark end of some part of simulation?
These were the first, basic things discussed on DASS.

**Review by raj-vlabs — CHANGES_REQUESTED** [review]
1. The pause button stays disabled throughout in this experiment. One can only start once and then just restart.
2. Please name variables meaningfully.

File: experiment/pretest.json (hunk 3 above)
Comment (Dineshg49) [reply experiment/pretest.json]: SME sent both the questions with the same answer. I had a doubt that the answer for this question might be wrong, so I confirmed SME, he agreed with me on this and I changed the answer in the same file. I didn't notice that question was repeated so failed to update the answer in the other file.

**Review by raj-vlabs — APPROVED** [review]

**Pull request merged**